packages/web3-eth-ens/src/abi/ens/DummyNameWrapper.ts

```diff
@@ -0,0 +1,40 @@
+/*
+This file is part of web3.js.
+
+web3.js is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+web3.js is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
+*/
+export const DummyNameWrapperApi = [
```

Comment:
If you could move this, please, to `packages/web3-eth-ens/test/fixtures` similar to how it is done in other packages. Like for example: `packages/web3-eth-contract/test/fixtures/erc721.ts`.

Reply:
In this package, the abis are used also in the class. The whole `ens` class is a wrapper of these smart contracts.

Reply:
Thanks for the clarification.
And so I hope to have a better name for that. Because the word "Dummy" is not very instructive. Or what do you think?

Reply:
You are right about that. The reason I gave that name, is to point to the right [contract](https://github.com/ensdomains/ens-contracts/tree/master/contracts/resolvers/mocks) (which I used) and avoid any misunderstandings. What do you think, should I rename it?

Reply:
Yes, please, as I think the name Dummy miss-leaded me a little :blush: